db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/utilMain.java

```diff
@@ -845,4 +820,26 @@ int getCurrentRowNumber(ResultSet rs)
 	public final Object run() {
 		return  getClass().getResourceAsStream(ProductGenusNames.TOOLS_INFO);
 	}
+	@SuppressFBWarnings(value = "RV_RETURN_VALUE_IGNORED_BAD_PRACTICE",justification = "Intentional, we don't care if the file already exists")
+	public void startSpooling(String path) throws IOException {
+		doSpool = true;
+		logFileName = path;
+		File f = new File(logFileName);
+		f.createNewFile(); // no-op if file exists.
+		this.out = new LocalizedOutput(new ForkOutputStream(new FileOutputStream(f)));
+	}
+
+	public void stopSpooling() {
+		doSpool = false;
+		this.out.flush();
+		this.out = new LocalizedOutput(System.out);
```

Comment:
We should close the FileOutputStream, relying on finalizers is very finicky

Reply:
ok